### PR TITLE
Add script to increase gas limit to 120M on mainnet

### PR DESCRIPTION
**Pull request by henridevieux**
Adds script to update the gasLimit on Base mainnet to 120M.

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: mainnet/.env [hunk 1]
```diff
@@ -1,5 +1,6 @@
 NETWORK=mainnet
 L1_RPC_URL=https://eth.llamarpc.com
+# L1_RPC_URL=https://c3-chainproxy-eth-base-mainnet-full.cbhq.net
```
Comment (0x00101010) [inline mainnet/.env:3]: remove this?

File: mainnet/2024-07-31-increase-gas-limit/README.md [hunk 2]
```diff
@@ -0,0 +1,292 @@
+# Update Gas Limit in L1 `SystemConfig` 
+
+Status: DONE
+
+## Objective
+
+We are updating the gas limit to improve TPS and reduce gas fees. 
+
+This runbook implements scripts which allow our signers to sign two different calls for our Incident Multisig: 
+1. `UpdateGasLimit` -- This script will update the gas limit to our new limit of 100.5M gas
```
Comment (0x00101010) [inline mainnet/2024-07-31-increase-gas-limit/README.md:10]:
```suggestion
1. `UpdateGasLimit` -- This script will update the gas limit to our new limit of 120M gas
```

File: mainnet/2024-07-31-increase-gas-limit/README.md [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+2. `RollbackGasLimit` -- This script establishes a rollback call in the case we need to revert to 97.5M gas
```
Comment (0x00101010) [inline mainnet/2024-07-31-increase-gas-limit/README.md:11]:
```suggestion
2. `RollbackGasLimit` -- This script establishes a rollback call in the case we need to revert to 112.5M gas
```

**Review by 0x00101010 — APPROVED** [review]

**Pull request merged**